src/command.ts

```diff
@@ -104,7 +104,13 @@ export default abstract class Command {
 
   static set enableJsonFlag(value: boolean) {
     this._enableJsonFlag = value
-    if (value) this.globalFlags = jsonFlag
+    if (value) {
```

Comment:
@mdonnalley For a subclass where base class enables json and then the subclass disables json, second time through (value === false) needs to be applied global.

Reply:
I think the following might be a cleaner solution.

1. Remove the getter/setter for `enableJsonFlag`. So now you just have a normal static class property

```
static enableJsonFlag = false
```

2. Move the json flag addition to `Command.parse` by replacing the assignment of `opts.flags` with the following
```
opts.flags = (this.ctor.enableJsonFlag ? {...options?.flags, ...jsonFlag} : options?.flags) as Interfaces.FlagInput<F>
```